pkgs/applications/science/math/cntk/default.nix

```diff
@@ -73,6 +76,9 @@ in stdenv.mkDerivation rec {
       ln -s ${cudnn}/include cuda
       export configureFlags="$configureFlags --with-cudnn=$PWD"
     ''}
+    
+    # Looks for openblas header by default, but we include compiled openblas
+    substituteInPlace ../configure --replace "include/openblas_config.h" "lib/libblas.so"
```

Comment:
This fixes, or at least proceeds past, the current issue on Hydra:
```
Invalid openblas directory /nix/store/xx0m1kwdmymicwlyydqh3xb6bxzm8kyy-openblas-0.3.13
```

The configure script expects [`include/openblas_config.h`](https://github.com/microsoft/CNTK/blob/ae9c9c7c5f9e6072cc9c94c254f816dbdc1c5be6/configure#L38) by default. Not sure how this was working prior.

Reply:
probably need to pass it `openblas.dev`

Reply:
if it needs both headers and library... then that will be more difficult

Reply:
Seems you are correct, removed this in favor of `openblas.dev`.